### ISSUE TITLE
ploting values over each country

### ISSUE THREAD

**Issue by AhmedElsayed-17** [erdogant/worldmap#4]
Since I'm a bit newbie in python, is it possible to plot values over each country? (GDP, population, etc).
These data are inside excel sheet, I just don't know how to place the values accurately over each country.

**Comment by erdogant** [comment]
Yes sure, on the documentation pages is an [example](https://erdogant.github.io/worldmap/pages/html/Examples.html).

**Comment by erdogant** [comment]
I guess, [this](https://d3blocks.github.io/d3blocks/pages/html/Maps.html) is what you are looking for.

**Comment by erdogant** [comment]
closing this one.
